source/libs/cache.ts

```diff
@@ -20,7 +20,7 @@ export async function getSet(key: string, getter: CacheGetter, expiration?: numb
 	}
 }
 
-export async function get(key: string) {
+export async function get(key: string): Promise<any> {
```

Comment:
Maybe
```suggestion
export async function get<T extends any>(key: string): Promise<T> {
```

Reply:
We can make this change, but from what I've seen in the code base `get` in JS land is always returning a `string` no? So we can type it as `Promise<T>` with the caveat that the string will be coerced to whatever type T is.

Reply:
> `get` in JS land is always returning a `string` no?

Which `get`? Why would it only return a string? `Map#get` doesn't 🤔

Reply:
Ahh, yeah, nevermind that comment. The only thing to alter is it should return `Promise<T | undefined>` because of 

```javascript
	// If it's not in the cache, it's best to return "undefined"
	if (value === null || value === undefined) {
		return undefined;
	}
```

Reply:
I think we're good to go after that.

Reply:
Actually, even if we type the get, it's possible undefined so you're not guaranteed a number in the case of the releases tab.

![image](https://user-images.githubusercontent.com/833231/56457650-f247ce80-634b-11e9-878c-bea47d0deb54.png)

I'm going to wrap this up tonight (just starting my day 😉), but feel free to jump on this in the meantime if you want.

Reply:
Ahh, I remember why I said it was always a string.

![image](https://user-images.githubusercontent.com/833231/56457700-52d70b80-634c-11e9-9c95-c70364915558.png)

I'm guessinf the `CacheGetter` interface should be generic, `T | Promise<T>` instead of returning `string | Promise<string>`?

Reply:
Ahh indeed. Values are stored in `document.cookie` so yes, they're always strings.